contents/docs/libraries/ios/index.mdx

```diff
@@ -36,6 +36,116 @@ PostHog supports the following Apple platforms:
+-   **Application Backgrounded** - when the app is sent to the background by the user
+-   **Application Installed** - when the app is installed.
+-   **Application Updated** - when the app is updated.
+-   **$screen** - when the user navigates (if using `UIViewController`)
+
+### Autocapture configuration
+
+You can enable or disable screen and application lifecycle autocapture through the `PostHogConfig` object.
+
+```swift
+let POSTHOG_API_KEY = "<ph_project_api_key>"
+// usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'
+let POSTHOG_HOST = "https://us.i.posthog.com"
+
+let config = PostHogConfig(apiKey: POSTHOG_API_KEY, host: POSTHOG_HOST)
+
+// capture screen views
+config.captureScreenViews = true 
+// capture application lifecycle events
+config.captureApplicationLifecycleEvents = true
+
+PostHogSDK.shared.setup(config)
+```
+
+### Capturing screen views
+
+With [`configuration.captureScreenViews`](/docs/libraries/ios#all-configuration-options) set as `true`, PostHog will try to record all screen changes automatically.
+
+If you want to manually send a new screen capture event, use the `screen` function.
+
+```swift
+PostHogSDK.shared.screen("Dashboard", properties: ["fromIcon": "bottom"])
+```
+
+> **Important:** Please note that `captureScreenViews` configuration is intented for `UIKit` use. For `SwiftUI` screen captures, please see the following section.
+
+### Capturing screen views in SwiftUI
+
+To track a screen view in `SwiftUI`, apply the `postHogScreenView` modifier to your full-screen views. The PostHog iOS SDK will infer a screen name based on the view’s type, but you can provide a custom name and event properties if needed.
+
+```swift 
+// This will trigger a screen view event with $screen_name: "HomeViewContent"
+struct HomeView: View {
+    var body: some View {
+        HomeViewContent()
+            .postHogScreenView()
+    }
+}
+
+// This will trigger a screen view event with $screen_name: "My Home View" and an additional event property from_button: "start"
+struct HomeView: View {
+    var body: some View {
+        HomeViewContent()
+            .postHogScreenView("My Home View", ["from_button": "start"])
+    }
+}
+```
+
+*In SwiftUI, views can range from entire screens to small UI components. Unlike UIKit, SwiftUI doesn’t clearly distinguish between these levels, which makes automatic tracking of full-screen views harder.*
+
+## Autocapture interactions
+
+> 🚧 **Note:** Autocapturing element interactions are currently only supported in `UIKit` and are in an experimental state (disabled by default). SwiftUI support will be coming soon.
+
+If enabled, PostHog will automatically track interactions with elements in a screen and track them as the following event: 
+
+-   **$autocapture** - interactions with elements in a screen (if using `UIKit`)
+
+### Autocapture interactions configuration
```

Comment:
I left a comment on the ios PR about the autocapture, take a look, this would likely change this PR if you agree with the suggestion.